classes/options-general.php

```diff
@@ -642,24 +642,24 @@ class='howto'><?php _e( 'Select "No Limit" to display all results within the sea
 													</tr>
 
 													<tr valign="top">
-														<td width="150"><?php _e( 'Google Adsense for Maps', 'simplemap' ); ?></td>
+														<td width="150"><?php _e( 'Google Adsense for Maps', 'SimpleMap' ); ?></td>
 														<td>
 															<label for="adsense_for_maps"><input type="checkbox"
 															                                     name="adsense_for_maps"
-															                                     id="adsense_for_maps" <?php checked( $adsense_for_maps ); ?> /> <?php _e( 'Enable Adense for Maps?', 'simplemap' ); ?>
+															                                     id="adsense_for_maps" <?php checked( $adsense_for_maps ); ?> /> <?php _e( 'Enable Adense for Maps?', 'SimpleMap' ); ?>
 															</label>
 															<br/><label for="adsense_pub_id">
-																<small><?php _e( 'Default Adsense Publisher ID:', 'simplemap' ); ?></small>
+																<small><?php _e( 'Default Adsense Publisher ID:', 'SimpleMap' ); ?></small>
 																<input type="text" name="adsense_pub_id"
 																       id="adsense_pub_id" size="30"
 																       value="<?php echo esc_attr( $adsense_pub_id ); ?>"/></label>
 															<br/><label for="adsense_channel_id">
-																<small><?php _e( 'Default Adsense Channel ID:', 'simplemap' ); ?></small>
+																<small><?php _e( 'Default Adsense Channel ID:', 'SimpleMap' ); ?></small>
```

Comment:
__[code_sniffer]__
* Spaces must be used for alignment; tabs are not allowed
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found
* Line exceeds 120 characters; contains 137 characters

<!-- Comment by [SideCI](https://sideci.com) -->